Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gableroux/unity3d:2018.3.14f1-android
+# install android sdk
+RUN touch /root/.android/repositories.cfg \
+    && /opt/android-sdk-linux/tools/bin/sdkmanager --install tools "platforms;android-28" "build-tools;28.0.3" "extras;android;m2repository" "platform-tools"
+RUN export PATH="/opt/android-sdk-linux/platform-tools/:$PATH"
```

Comment:
Its best to avoid multiple RUN instructions and chain commands together when possible, each instruction create another layer, so the image size gets much bigger

https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better